nekoyume/Assets/_Scripts/UI/Widget/Popup/AdventureBossNcgRandomRewardPopup.cs

```diff
@@ -30,6 +30,12 @@ public void Show(long seasonId, bool ignoreShowAnimation = false)
         public async UniTaskVoid ShowWinner(long seasonId)
         {
             var explorerBoard = await Game.Game.instance.Agent.GetExploreBoardAsync(seasonId);
+            if(explorerBoard.RaffleWinnerName == string.Empty)
```

Comment:
```suggestion
            if(string.IsNullOrEmpty(explorerBoard?.RaffleWinnerName))
```
아마 동작 차이는 없을 것 같지만 이쪽이 조금 더 안전해보이는 기분이 드는 것 같기도

Reply:
오 확실히 더 안전해 보입니다~! 감사합니다~!